examples/3d/motion_blur.rs

```diff
@@ -155,33 +155,33 @@ fn spawn_cars(
                 },
                 Moves(i as f32 * 2.0),
             ))
-            .insert_if(CameraTracked, || i == 0);
-        entity.with_children(|parent| {
-            parent.spawn(PbrBundle {
-                mesh: box_mesh.clone(),
-                material: color,
-                transform: Transform::from_xyz(0.0, 0.08, 0.03)
-                    .with_scale(Vec3::new(1.0, 1.0, 0.5)),
-                ..default()
+            .insert_if(CameraTracked, || i == 0)
+            .with_children(|parent| {
```

Comment:
Only indentation/whitespace changes below this comment in this file :)

gits diffs do a surprisingly bad job with indentation changes